gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -309,7 +309,7 @@ def _update_google_cloud_job_name(google_cloud_options, job_name):
 
 
 def _merge_headers(known_args, pipeline_args,
-                   pipeline_mode, annotated_vcf_pattern=None):
+                   pipeline_mode, avro_root_path, annotated_vcf_pattern=None):
```

Comment:
In this method, we create sample info table (line 335). We should move it to where we create all other tables before AVRO load step.

Reply:
Umm, disagree - we have headers here, we would need to recreate them, which would be an unnecessary computation. WDYT?